src/coprocessor/dag/expr/ctx.rs

```diff
@@ -46,6 +20,71 @@ bitflags! {
     }
 }
 
+bitflags! {
+    /// Flags are used by `DAGRequest.flags` to handle execution mode, like how to handle
+    /// truncate error.
+    pub struct Flags: u64 {
+        /// `FLAG_IGNORE_TRUNCATE` indicates if truncate error should be ignored.
+        /// Read-only statements should ignore truncate error, write statements should not ignore
+        /// truncate error.
+        const FLAG_IGNORE_TRUNCATE = 1;
```

Comment:
We can remove the FLAG prefix for simplicity.

Reply:
+1